posixlib/src/main/resources/scala-native/nativethread.c

```diff
@@ -0,0 +1,66 @@
+#include <sys/types.h>
+#include <stdint.h>
+#include <pthread.h>
+#include <sched.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+pthread_attr_t PTHREAD_DEFAULT_ATTR;
+struct sched_param PTHREAD_DEFAULT_SCHED_PARAM;
+int PTHREAD_DEFAULT_POLICY;
+size_t PTHREAD_DEFAULT_STACK_SIZE;
+int initialized = 0;
+
+void init() {
+    pthread_attr_init(&PTHREAD_DEFAULT_ATTR);
+    pthread_attr_getschedparam(&PTHREAD_DEFAULT_ATTR,
+                               &PTHREAD_DEFAULT_SCHED_PARAM);
+    pthread_attr_getschedpolicy(&PTHREAD_DEFAULT_ATTR, &PTHREAD_DEFAULT_POLICY);
+    pthread_attr_getstacksize(&PTHREAD_DEFAULT_ATTR,
+                              &PTHREAD_DEFAULT_STACK_SIZE);
+
+    initialized = 1;
+}
+
+int get_max_priority() {
```

Comment:
For helper methods. the standard is to prefix the function or define with `scalanative_`. Unfortunately, this was just documented and has not made it in the latest docs yet.
See commit: https://github.com/scala-native/scala-native/pull/2159/commits/4f5859a64e2d5dbcd9b5ade5ab2fd6525552c24a

Reply:
@WojciechMazur Is it easy enough to publish the latest? I am wondering if a link somewhere pointing to the latest might be good for contributors and people that are curious?